### ISSUE TITLE
entrez-direct does not have help pages setup correctly

### ISSUE THREAD

**Issue by arteymix** [bioconda/bioconda-recipes#31307]
```bash
env LANG=C esearch --help
esearch 15.6

cat: /home/guillaume/miniconda3/bin/help/esearch-help.txt: No such file or directory
```

**Comment by ialbert** [comment]
Sorry about that, I have packaged that, it looks like it needs to copy some text files to the bin folder. I'll look into it.

**Comment by ialbert** [comment]
https://github.com/bioconda/bioconda-recipes/pull/31397

**Comment by ialbert** [comment]
has been merged and fixed with version 16.2

**Comment by arteymix** [comment]
I can confirm that it works now!
